### ISSUE TITLE
Review training/evaluation implementation

### ISSUE THREAD

**Issue by tschaffter** [data2health/covid19-challenge#89]
Initially in Q1, there is no training.

- If workflow is configured with training name set to empty or training version is empty, do not run training

**Comment by thomasyu888** [comment]
Also how should we handle those questions that allows training?  Are we going to require people to have an `/app/train.sh` for those even if it is just empty?

How will those people know if their training has failed?

**Comment by tschaffter** [comment]
## Update

Infra improvements:

- [x] Add LABEL `enable_train = TRUE` to enable training (variable name to be updated)
  - Not required, if not specified, assume FALSE
- [x] If there is NO training dataset
  - Do not attempt to train
  - Set training dataset version to blank ("")
- [x] If there is a training dataset, then
  - Look for `enable_train = TRUE`, then
  - Train
- [x] If training fails for any reason, terminate submission
- [x] During training, we mount only the training set
- [x] During evaluation, we mount only the evaluation set
- [x] Update past submissions
  - Training dataset version to be set to ""
